app/grandchallenge/emails/views.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.views.generic import CreateView, DetailView, UpdateView
+from guardian.mixins import LoginRequiredMixin
+
+from grandchallenge.emails.forms import EmailForm
+from grandchallenge.emails.models import Email
+
+
+class EmailCreate(
+    LoginRequiredMixin,
+    PermissionRequiredMixin,
+    CreateView,
+):
+    model = Email
+    form_class = EmailForm
+    permission_required = "emails.add_email"
+    raise_exception = True
+
+
+class EmailUpdate(
+    LoginRequiredMixin,
+    PermissionRequiredMixin,
+    UpdateView,
+):
+    model = Email
+    form_class = EmailForm
+    permission_required = "emails.change_email"

```

Comment:
We should also check that the email has not been sent, maybe raise a permission denied if it has?